mlflow/server/js/src/experiment-tracking/components/ExperimentRunsTableMultiColumnView2.js

```diff
@@ -335,6 +335,13 @@ export class ExperimentRunsTableMultiColumnView2 extends React.Component {
     this.applyRowSelectionFromProps();
     this.handleColumnSizeRefit();
     this.handleLoadingOverlay();
+
+    window.addEventListener(
+      'resize',
+      _.debounce(() => {
+        this.gridApi.sizeColumnsToFit();
+      }, 100),
```

Comment:
Wrapped `this.gridApi.sizeColumnsToFit()` with an arrow function to avoid an error caused by javasript's what-"this"-refers-to issue.

![Screen Shot 2020-07-28 at 18 16 44](https://user-images.githubusercontent.com/17039389/88645092-b2e76080-d0fe-11ea-9f26-ddd470e41075.png)

![Screen Shot 2020-07-28 at 18 18 57](https://user-images.githubusercontent.com/17039389/88645205-d4e0e300-d0fe-11ea-8bbe-8d2bf5e452fa.png)